cli/build.rs

```diff
@@ -153,10 +153,63 @@ mod ts {
     #[op]
     fn op_build_info(state: &mut OpState) -> Value {
       let build_specifier = "asset:///bootstrap.ts";
+      // todo(dsherret): this is a temporary solution as we should have a
+      // single source of truth for this stored in the Rust code that we
+      // send to TypeScript, but from cli/build.rs we can't access anything
+      // in the CLI crate. When refactoring that code out of the CLI, then
+      // we can use that information here.
+      let node_built_in_module_names = vec![
```

Comment:
I suggest we move `NodeModulePolyfill` and `SUPPORTED_MODULE` from `cli/node/mod.rs` to `ext/node/` - that way we should be able to use them here.